velox/exec/tests/TableWriteTest.cpp

```diff
@@ -58,10 +60,11 @@ TEST_F(TableWriteTest, scanFilterProjectWrite) {
     writeToFile(filePaths[i]->path, vectors[i]);
   }
 
-  auto outputDirectory = TempDirectoryPath::create();
-
   createDuckDbTable(vectors);
 
+  HiveNoCommitWriteProtocol::registerProtocol();
```

Comment:
Can this be moved to SetUp? perhaps, even HiveConnectorTestBase::SetUp.

Should we test other commit strategies as well?

Reply:
One unit test is added for a second commit strategy.